index.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>Pizza</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link href="css/reset.css" rel="stylesheet">
+	<link href="css/main.css" rel="stylesheet">
+
+	<script src="js/main.js"></script>
+</head>
+<body onload="digitalWatch()">
+	<header>
+		<div class="header-wrapper">
+			<div id="digital_watch" class="watch-wrapper">
+			</div>
+		</div>
+		<div class="header-wrapper">
+			<a href="#" class="col-wrapper">
+				<div class="hvr-wobble-top"></div>
+				<p>Pizza Manager</p>
+			</a>
+		</div>
+		<div class="header-wrapper">
+			<div class="col-wrapper col-wrapper-grid">
+				<button>SING IN</button>
+				<button>SING UP</button>
+			</div>
+		</div>
+	</header>
+
+	<content>
+		<button class="add-pizza">+ ADD NEW PIZZA</button>
+		<div class="container">
+			<a href="#">
```

Comment:
I don't see why each pizza order should be wrapped by anchor `<a>` element

Reply:
I most cases when you develop cards they must be clickable.

Card is like big button, with image, text and some buttons. There not only buttons must be clickable but other space, especially image, must too.